### PR TITLE
[XLink] - run XLink tests in pre-commit

### PR DESCRIPTION
**Pull request by DariaMityagina**
### Details:
Purpose: run XLink tests to validate the PRs that update the XLink code from the OpenVINO repository.

### Tickets:
-76384

### CODE REVIEW

File: .ci/azure/linux.yml [hunk 1]
```diff
@@ -288,6 +288,10 @@ jobs:
     displayName: 'VPU UT'
     continueOnError: false
 
+  - script: . $(SETUPVARS) && $(REPO_DIR)/src/plugins/intel_myriad/third_party/tests/ --gtest_output=xml:TEST-XLinkTests.xml
```
Comment (azhogov) [inline .ci/azure/linux.yml:291]: tests should be installed

File: .ci/azure/windows.yml [hunk 2]
```diff
@@ -240,6 +240,10 @@ jobs:
     displayName: 'VPU UT'
     continueOnError: false
 
+  - script: call $(SETUPVARS) && $(REPO_DIR)/src/plugins/intel_myriad/third_party/tests/ --gtest_output=xml:TEST-XLinkTests.xml
```
Comment (azhogov) [inline .ci/azure/windows.yml:243]: install tests

**Review by azhogov — CHANGES_REQUESTED** [review]

File: .ci/azure/linux.yml (hunk 1 above)
Comment (DariaMityagina) [reply .ci/azure/linux.yml]: done, thank you!

File: .ci/azure/windows.yml (hunk 2 above)
Comment (DariaMityagina) [reply .ci/azure/windows.yml]: done, thank you!

**Comment by DariaMityagina** [comment]
@azhogov , @beleiuandrei , @Maxim-Doronin could you please take a look?

**Review by beleiuandrei — APPROVED** [review]

**Review by azhogov — APPROVED** [review]

**Pull request merged**